tests/acceptance/features/webUISharingInternalUsers/shareAutocompletion.feature

```diff
@@ -16,7 +16,6 @@ Feature: Autocompletion of share-with names
       | username  | password  | displayname     | email          |
       | two       | %regular% | User Two        | u2@oc.com.np   |
       | u444      | %regular% | Four            | u3@oc.com.np   |
-      | five      | %regular% | User Group      | five@oc.com.np |
```

Comment:
why deleting this line?

Reply:
Since it's possible now to search with username as well, the user five is found also when typing "fi" into autocomplete but displaying as "User Group" so it doesn't meet the condition of all the results containing "fi" thus leading to a failed test.